sdk/keyvault/azure-security-keyvault-certificates/pom.xml

```diff
@@ -119,5 +110,38 @@
       <version>4.11.0</version> <!-- {x-version-update;org.mockito:mockito-core;external_dependency} -->
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.bouncycastle</groupId>
```

Comment:
While this is a test dependency, not a fan of bouncy castle, due to its CVE issues and can influence users to use Bouncy castle as the code is tested with it.

As a follow up investigation, would be good to replace with an alternative library.